src/ows/src/main/java/org/geoserver/ows/Dispatcher.java

```diff
@@ -1151,6 +1151,10 @@ Service findService(String id, String ver, String namespace) throws ServiceExcep
         if (matches.size() > 1) {
             List vmatches = new ArrayList(matches);
 
+            if (isCiteCompliant() && version == null && "WFS".equals(id)) {
```

Comment:
@nmco is just working on making WFS 2.0 pass CITE tests... so maybe avoid disabling it fully? Maybe we'll need some way to selectively disable versions (would be useful for WCS CITE testing as well)